models/accreditation.js

```diff
@@ -0,0 +1,14 @@
+import connector from "#models/databaseUtil";
+
+const accreditationSchema = {
+    uid :{type : String, unique: true , required: true},
+    accreditationName: {type: String,   required: true}, 
+    agencyName: {type: String,  required: true},
+    dateofAccreditation: {type: Date,   required: true},
+    dateofExpiry: {type: Date, required: true}
+  
+}
+
+const AccreditationSchema = new connector.model('Accreditation_schema', accreditationSchema);
+ 
+export default accreditationSchema;
```

Comment:
You're exporting the wrong object, you need to export AccreditationSchema instead. Also, just to avoid confusions, please rename it to AccreditationModel.